src/node/rpc/frontend.h

```diff
@@ -503,7 +503,7 @@ namespace ccf
           jsonrpc::error_response(
             0,
             jsonrpc::CCFErrorCodes::INVALID_CALLER_ID,
-            "No corresponding caller entry exists."),
+            "Could not find matching user certificate."),
```

Comment:
This is also called by members, where this error is incorrect. We could use `callers->get_name()`, but I think that will be `ccf.user_certs`?

Reply:
actor certificate?

Reply:
Maybe overkill, but as a general solution:

`frontend.h`:
```
virtual std::string invalid_caller_error_message() const {
  return "Could not find matching actor certificate";
}
```

`memberfrontend.h`:
```
std::string invalid_caller_error_message() const override {
  return "Could not find matching member certificate";
}
```

etc